benchmark.js

```diff
@@ -12,6 +12,8 @@ var Benchmark = (function() {
     // to take effect, unless the distribution is too weird
     numRounds: 30,
     roundDelay: 5000, // ms to delay starting next round of tests
+    warmBench: false,
```

Comment:
As with *deleteFs* and *deleteJitCache*, *warmBench* should show up in the *config* output. Although I'm unsure why it's configurable. Is it because it takes longer to run?